libfreerdp/core/peer.c

```diff
@@ -122,6 +123,7 @@ static BOOL freerdp_peer_virtual_channel_close(freerdp_peer* client, HANDLE hCha
 	mcsChannel = peerChannel->mcsChannel;
 	WINPR_ASSERT(mcsChannel);
 	mcsChannel->handle = NULL;
+	DeleteCriticalSection(&peerChannel->writeLock);
```

Comment:
Same here: In this commit (`[core,server] lock FreeRDP_WTSVirtualChannelWrite`), this code here is still there from the old version. This should be removed, as it would break bisecting otherwise (fails to compile).